### ISSUE TITLE
BC Fossil Storage Location and BC Fossil Type - questions and mandatory attributes to mark

### ISSUE THREAD

**Issue by LeannePyle** [bcgov/BCHeritage#440]
BC Fossil Storage Location – re-order

- [x] Start with Storage Location Identifier
- [x] Location Name and Location Code – mark as mandatory
- [x] Storage Location Place
- [x] Storage Location Place – make just address mandatory
- [x] Location Primary Email
- [x] Storage Location Notes

BC Fossil Type

- [x] reorder to have Fossil Name first
- [ ] Rule – if “Common” is selected, then following fields are hidden BF: This is dependent on #470. Lets push this out as a separate issue
- [x] Fossil Name – mark as mandatory
- [x] Name type - mark as mandatory
- [x] Size Category - mark as mandatory
- [x] Fix Parent Genus LOV

--when is someone entering something in here? BF: I see this as a FMO managed list. It is used to define the controlled vocabulary (LOV) for species found in collected samples.
--what is parent name pointing to? - BF: Parent name is the parent in the Taxonomic Rank hierarchy. Originally named "Parent" because I believed we were going to be using more of the hierarchy than just Genus & Species. Need to decide how much of the Taxonomic hierarchy will be implemented and rename this field accordingly.

@LeannePyle  - For Storage Locations:
- [x] Should we change the "Storage Location Place" name to be "Contact information" and put the email address on this card? - BF - Yes as per 2023-06-14 meeting discussion
- [x] Do we need the Address Type field? - BF - Remove as per 2023-06-14 meeting discussion
- [x] Should we split the address into the typical fields (Address Line 1 & 2, City, Province, Postal Code) - Copy BCRHP BC Property Address card. Change first two to Address Line 1 & 2
- [x] Should we get rid of the Geometry Qualifier? - BF - Yes as per 2023-06-14 meeting discussion
- [x] Do we need the "Note Type" field? - BF - Remove as per 2023-06-14 meeting discussion
